jobs/update-colin-filings/update_colin_filings.py

```diff
@@ -86,18 +86,18 @@ def get_filings(app: Flask = None):
     return r.json()
 
 
-def send_filing(app: Flask = None, filing: dict = None):
+def send_filing(app: Flask = None, filing: dict = None, filing_id: str = None):
     """Post to colin-api with filing."""
-    clean_none(filing)
+    # clean_none(filing)
```

Comment:
Is this supposed to be commented out?

Reply:
Yep! Commenting it out because I don't think it's needed anymore, but I didn't want to delete the code in case we need it for a use case I'm not thinking of